mail-client/claws-mail/claws_mail-4.1.0.recipe

```diff
@@ -0,0 +1,164 @@
+	app:Claws_mail$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	xcairo$secondaryArchSuffix
+	gtk3$secondaryArchSuffix
+	lib:libarchive$secondaryArchSuffix
+	lib:libatk_1.0$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libdb_6.0$secondaryArchSuffix
+	lib:libenchant_2$secondaryArchSuffix
+	lib:libetpan$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgpgme$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	lib:libgumbo$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libical$secondaryArchSuffix
+	lib:libICE$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:liblber$secondaryArchSuffix
+	lib:libldap$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libnettle$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libpoppler_glib$secondaryArchSuffix
+	lib:librsvg_2$secondaryArchSuffix
+	lib:libsasl2$secondaryArchSuffix
+	lib:libSM$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	xcairo${secondaryArchSuffix}_devel
+	gtk3${secondaryArchSuffix}_devel
+	devel:libarchive$secondaryArchSuffix
+	devel:libatk_1.0$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libdb_6.0$secondaryArchSuffix
+	devel:libenchant_2$secondaryArchSuffix
+	devel:libetpan$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libgdk_pixbuf_2.0$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libgpgme$secondaryArchSuffix
+	devel:libgthread_2.0$secondaryArchSuffix
+	devel:libgumbo$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libical$secondaryArchSuffix
+	devel:libICE$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:liblber$secondaryArchSuffix
+	devel:libldap$secondaryArchSuffix
+	devel:libgmodule_2.0$secondaryArchSuffix
+	devel:libnettle$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libpangocairo_1.0$secondaryArchSuffix
+	devel:libpoppler_glib$secondaryArchSuffix
+	devel:librsvg_2$secondaryArchSuffix
+	devel:libsasl2$secondaryArchSuffix
+	devel:libSM$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:bison
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
```

Comment:
The dependency on cmd:git should be removed.

Reply:
That's a bit tricky, because without it the build remains broken,  and before adding it i searched in other recipes and found that others used cmd:git too, so i thought it is just a cmd like any other?
I left a comment where i used it in the BUILD() section.
Another solution is to build it without creating a git repo, but i don't know how to do that from within the recipe.

Reply:
Well, using git to get the version is incorrect anyway. So it means you have to disable the .git directory check, so that the file version is used to get the version, or set a variable with $portVersion.

Reply:
That does not explain why the cmd:git should be removed?, while atleast 20+ other recipes include it.

Reply:
I just explained why.

Reply:
I am merely providing this PR as a courtesy for the non power users in the community, personally i don't need it i have it running on my system, you can do whatever you want with it you can even close it if that will make you feel  good ,I'm done replying about the "cmd:git" thing which is clearly just a whim.

Reply:
@pulkomandy  My one liner solution addresses exactly all what you've said, that is it only restores the original version release that is being built after it has been altered by git init & ./configure, the other proposed solution was to patch a paragraph of 9 lines or so in configure.ac making an ugly patch and does not restore the correct version.